SG-35164 Add support for 3dsmax 2025 (#41)
shotgunsoftware/tk-3dsmax: docs/index.rst

This engine supports 3dsMax version 2017 and up. It favors the use of ``pyxms`` over ``MaxPlus`` and will be the version of the engine that will be maintained in the future.

> If you are reading this and are still using the ``tk-3dsmaxplus`` version or the original ``tk-3dsmax`` version that uses the ``Blur Python`` package, you should consider upgrading as we deprecated both the previous ``tk-3dsmax`` implementation and the ``tk-3dsmaxplus`` implementation.

Reviewer: ```suggestion
If you are reading this and are still using the ``tk-3dsmaxplus`` version or the original ``tk-3dsmax`` version that uses the ``Blur Python`` package, please note that these implementations are deprecated and no longer supported.
```
